### PR TITLE
Kiali v0.5.0 on Ansible Installer

### PR DESCRIPTION
**Pull request by gbaufake**
Hi Istio Team,

As #6929, I am sending this PR to update Kiali (v0.5.0) on Ansible Installer as well

More information about Kiali v0.5.0 - https://medium.com/kialiproject/kiali-sprint-7-istio-0-8-support-4df62e5291f2

Best Regards,
Guilherme Baufaker Rêgo

### CODE REVIEW

**Comment by gbaufake** [comment]
cc @sdake @gyliu513 @ymesika

File: install/ansible/istio/vars/main.yml [hunk 1]
```diff
@@ -14,4 +14,5 @@ istio_simple_samples:
   - name: sleep
     path: sleep/sleep.yaml
 
-istio_k8s_install_path: "install/kubernetes"
\ No newline at end of file
+istio_k8s_install_path: "install/kubernetes"
+kiali_version: v0.5.0
```
Comment (gyliu513) [inline install/ansible/istio/vars/main.yml]: why not keep this in `install_addons.yml`?

Comment (gbaufake) [reply install/ansible/istio/vars/main.yml]: I removed from that just to make all vars on the same place.
Should I return to `install_addons.yml`?

Comment (gyliu513) [reply install/ansible/istio/vars/main.yml]: I see and I'm ok to move this to same place. 

Same as https://github.com/istio/istio/pull/6985 , as master is frozen, can you please create PR against 1.0 branch?

**Comment by gbaufake** [comment]
closing in favor of #7042

**Pull request closed without merging**